contracts/UniswapV3Staker.sol

```diff
@@ -103,10 +103,18 @@ contract UniswapV3Staker is IUniswapV3Staker, Multicall {
             block.timestamp <= key.startTime,
             'UniswapV3Staker::createIncentive: start time must be now or in the future'
         );
+        require(
+            key.startTime - block.timestamp <= 365 days,
```

Comment:
maybe put these in immutables so we can change them post-audit without changing the code